guides/user/triggers/webhooks/index.md

```diff
@@ -0,0 +1,149 @@
+---
+layout: single
+title:  "Triggering on Webhooks"
+sidebar:
+  nav: guides
+---
+
+{% include toc %}
+
```

Comment:
The configuration for a GitHub trigger is maybe a bad example because it is configured entirely differently, but I can point out that these triggers don't have to come from your command line

Reply:
rewritten and made the third-party vs your own payload distinction a little clearer